### PR TITLE
Create the JAR to wrap the actual gw.exe process.

### PR DESCRIPTION
**Pull request by JunyiYi**

### CODE REVIEW

File: dist_pkgs/maven/iot-gateway-win32/pom.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.microsoft.azure.gateway</groupId>
```
Comment (jeffreyCline) [inline dist_pkgs/maven/iot-gateway-win32/pom.xml]: Should this be Microsoft.Azure.Devices.Gateway instead of microsoft.azure.gateway?

Comment (jeffreyCline) [inline dist_pkgs/maven/iot-gateway-win32/pom.xml]: Should this be Microsoft.Azure.Devices.Gateway instead of microsoft.azure.gateway?

Comment (JunyiYi) [inline dist_pkgs/maven/iot-gateway-win32/pom.xml]: Java has a convention that all namespaces are lower-case.
For reference: https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.microsoft.azure%22%20AND%20a%3A%22azure-documentdb%22

**Comment by jeffreyCline** [comment]
The casing is fine, I was speaking about the missing word devices between azure and gateway.

**Comment by JunyiYi** [comment]
Good point, I got the namespace "com.microsoft.azure.gateway.core" from their code "https://github.com/shsushi/azure-iot-gateway-sdk/blob/master/bindings/java/gateway-java-binding/src/main/java/com/microsoft/azure/gateway/core/IGatewayModule.java". We need to talk to Azure team to unify it.

**Pull request merged**